requirements.txt

```diff
@@ -0,0 +1 @@
+git+git://github.com/david415/python-ftools.git#egg=ftools
```

Comment:
I do not see it - https://pypi.python.org/pypi?%3Aaction=search&term=ftools&submit=search 
:(